src/test/java/org/jenkinsci/remoting/engine/JnlpProtocolHandlerTest.java

```diff
@@ -155,7 +155,7 @@ public void tearDown() {
     }
 
     @Theory
-    public void happyPath(Factory factory, boolean useNioHubServer, boolean useNioHubClient) throws Exception {
+    public void happyPath(Factory factory, boolean useNioHubServer, boolean useNioHubClient) throws Throwable {
```

Comment:
I don't understand the motivation for this change.  Can you explain further why the throw was broadened from Exception to Throwable?  The [stackoverflow article](https://stackoverflow.com/questions/2274102/difference-between-using-throwable-and-exception-in-a-try-catch) indicates that Throwable is broader than Exception, but I don't understand why a caller would need to know that this Theory may throw expections which are Throwable but are not Exceptions.

Reply:
My IDE was complaining about it, I don't mind reverting it

Reply:
Reverting that would be better. I don't see that it needs to widened to Throwable.

Reply:
Looks like you missed changing this one back.

Reply:
Now I see why you needed to change this from an Exception to a Throwable. This change [here](https://github.com/jenkinsci/remoting/pull/369/files#diff-9138943fb30c237d96a6976f2eab1105R40) changes the signature of this line [here](https://github.com/jenkinsci/remoting/blob/621b672f67ba595d9fd73f5fd748e0ea2e56cbe6/src/test/java/org/jenkinsci/remoting/engine/JnlpProtocolHandlerTest.java#L167) so that it now throws Throwable rather than Exception. Since it's test code, it should be okay changing the signature.

We've given you some bad advice about changing it back to Exception because we didn't understand how that other change ripples through.

This is why the build is failing.

Reply:
Oh well, It's all good.

The inheritance change is probably for the best so the class will advertise its exception and return type correctly